docs/getting-started/integrating-codacy-with-intellij-ides.md

```diff
@@ -0,0 +1,92 @@
+# Integrating Codacy with IntelliJ IDEs
+
+<!-- TODO ALA-872
+- review entire page for accuracy
+-->
+
+!!! info "To use this plugin you need a [Codacy account](https://www.codacy.com/signup-codacy)"
```

Comment:
The issue here is that the official name of IntelliJ add-ons is [plugin](https://www.jetbrains.com/help/idea/managing-plugins.html), and VS Code add-ons are [extensions](https://code.visualstudio.com/docs/editor/extension-marketplace).

Same thing, different terminology.

Since each respective user will be used to their IDE's terminology, I'd rather reflect that by using two different terms on the two pages.

OK about moving the admonition.